presto-product-tests/src/main/resources/sql-tests/testcases/window_functions/windowDescriptionAABwithEP.sql

```diff
@@ -0,0 +1,8 @@
+-- database: presto; groups: window;
+select
```

Comment:
If circumstances change (e.g. projection node is not there anymore) we'll need to change tests anyway (because they might not be valid anymore).